### PR TITLE
Update EIP-4750: Disallow dynamic jumps

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

File: EIPS/eip-4750.md [hunk 1]
```diff
@@ -116,17 +116,19 @@ In addition to container format validation rules above, we extend code section v
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
 
+### Deprecated instructions
+
+Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes do not have any instruction assigned.
```
Comment (chfast) [inline EIPS/eip-4750.md:121]:
```suggestion
Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes are undefined.
```

Not sure this is better, but I like "undefined opcode" and/or "undefined instruction" term.

**Review by chfast — COMMENTED** [review]
Looks good to me, but:

1. We need rationale that jumpdest analysis is not needed any more.
2. `NOP` change must be discussed.

File: EIPS/eip-4750.md (hunk 1 above)
Comment (axic) [reply EIPS/eip-4750.md]: Undefined sounds fine.

File: EIPS/eip-4750.md [hunk 2]
```diff
@@ -116,17 +116,19 @@ In addition to container format validation rules above, we extend code section v
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
 
+### Deprecated instructions
```
Comment (axic) [inline EIPS/eip-4750.md:119]: It uses "deprecated" in abstract, "disallowed" in motivation, "deprecated" here. We should choose one, and "disallowed" sounds closer to what it is.

**Comment by axic** [comment]
@gumb0 rebased to resolve conflicts.

File: EIPS/eip-4750.md (hunk 2 above)
Comment (gumb0) [reply EIPS/eip-4750.md]: Changed to "disallowed" everywhere

**Pull request merged**